runtime/libs/circle-schema/include/circle_traininfo.fbs

```diff
@@ -1,6 +1,6 @@
-// This file is from https://github.sec.samsung.net/one-project/circle-x/blob/e94b2b8956bcc316bef19dc4885a5027f545845a/spec/circle_training.fbs
+// This file is from https://github.sec.samsung.net/one-project/circle-x/blob/5dd9665b3dab4652a0afed50da86bf6964a41bdd/spec/circle_training.fbs
```

Comment:
### background

This PR is to sync with inner source schema - which is lately updated: https://github.sec.samsung.net/one-project/circle-x/commit/5dd9665b3dab4652a0afed50da86bf6964a41bdd 

### Notes 
After this syncing, I will remove the schema file in inner source and remove this commit number here.
I have a plan to maintain `circle_traininfo.fbs` here, not in inner source.  
(related inner source issue [:link:](https://github.sec.samsung.net/one-project/circle-x/issues/57)